website_docs/manual.md

```diff
@@ -7,9 +7,88 @@ aliases: [/docs/instrumentation/go/instrumentation, /docs/instrumentation/go/man
 
 Instrumentation is the process of adding observability code to your application. There are two general types of instrumentation - automatic, and manual - and you should be familiar with both in order to effectively instrument your software.
 
+## Initializing a Tracer
+
+To create spans, you'll need to initialize a tracer first. Some important things to consider:
+
+* Tracers should live at the module level and have a lifecycle congruent to an app's lifecycle
```

Comment:
If you mean `Tracer` here, they should generally be package-scoped and not module-scoped.  The lifecycle of a `Tracer` is more dependent on its associated `TracerProvider` than anything else, even though there isn't a clear mechanism for a stopped `TracerProvider` to stop a `Tracer`.  The `Tracer` will simply cease to create exported spans.